meetings/conferences/tests.py

```diff
@@ -1,38 +1,143 @@
-from django.test import TestCase
-from unittest import skip
+from django.test import TestCase, RequestFactory
+import factory
+from conferences.models import Conference
+from approvals import models as approvalModels
+from submissions import models as submissionModels
+from django.contrib.auth.models import User
+from permissions import ConferencePermissions
+from serializers import ConferenceSerializer
+from views import ConferenceViewSet
+
+
+class UserFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = User
+
+
+class ConferenceFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Conference
+
+
+class ApprovalFactory(factory.DjangoModelFactory):
```

Comment:
Could I just make a file under meetings called factories.py and import all of the factories from there?